frame/staking/src/lib.rs

```diff
@@ -955,11 +992,19 @@ pub mod pallet {
 	#[pallet::getter(fn validators)]
 	pub type Validators<T: Config> = StorageMap<_, Twox64Concat, T::AccountId, ValidatorPrefs, ValueQuery>;
 
+	/// A tracker to keep count of the number of items in the `Validators` map.
+	#[pallet::storage]
+	pub type ValidatorsCount<T> = StorageValue<_, u32, ValueQuery>;
```

Comment:
```suggestion
	pub type CurrentValidatorsCount<T> = StorageValue<_, u32, ValueQuery>;
```

Suggestion. As noted I think this can be mistaken with the other one.